fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidQueryBuilder.java

```diff
@@ -367,8 +370,9 @@ protected TimeSeriesQuery buildTimeSeriesQuery(
 
         LOG.trace("Building a single pass druid timeseries query");
 
+        // TODO FIXME hardcoding to concrete for now
         // The data source is the table directly, since there is no nested query below us
-        DataSource dataSource = new TableDataSource(table);
+        DataSource dataSource = new TableDataSource((ConcretePhysicalTable) table);
```

Comment:
There's a little tech debt here.  Existing union data sources will construct with either non-Concrete physical tables or sets of tables.  Table data source will only be usable by concrete physical tables.  This polymorphism will push up the stack, but I'd like to take it as tech debt right now.